database/migration/src/main/resources/changesets/enlarge_call_caching_hash_entry_id.xml

```diff
@@ -11,6 +11,10 @@
                 tableName="CALL_CACHING_HASH_ENTRY"
                 columnName="CALL_CACHING_HASH_ENTRY_ID"
                 newDataType="BIGINT"/>
+        <addAutoIncrement columnName="CALL_CACHING_HASH_ENTRY_ID"
```

Comment:
FYI- we may need to add this to a separate changeset. One of the potential sins 👿 discovered during #4919 is that `addAutoIncrement` seemed to cause an error in PostgreSQL, even though [the liquibase docs say it is supported](https://www.liquibase.org/documentation/changes/add_auto_increment.html).

However, that ongoing PR already has multiple `addAutoIncrement` issues to deal with, so this addition just be a +1 of things to fix.